model/src/main/proto/topic.proto

```diff
@@ -127,3 +127,7 @@ message ChapterProgress {
   // Corresponds to whether this chapter is playable or has been started.
   ChapterPlayState play_state = 2;
 }
+
+message CellularDataPreference {
+  bool hideDialog = 1;
```

Comment:
Please add comments for the message & field to explain what it's for.

Proto field names should be in snake_case.

Regarding the name, it made me think that this structure might be incomplete. Don't we need to store two things:
- Whether to show the dialog again?
- In the case where we shouldn't show the dialog, whether to play audio on cellular?

Reply:
Yeah I think it's needed too.